### DISCUSSION TITLE
Post-processing for AMOS22

### DISCUSSION THREAD

**Discussion by function2-llx** [MIC-DKFZ/nnUNet#1308] [Q&A] [answered]
Hi there,

Thanks again for sharing the code for AMOS22. When I look into the inference code, there are plenty of post-processing procedures I've never seen before. Could you please give some hints on what's going on during post-processing or give some reference? Thanks a lot!

Best

**Comment by FabianIsensee** [comment, marked as answer]
Have you read the corresponding section in our paper? 
https://arxiv.org/pdf/2208.10791.pdf

**Reply by function2-llx** [reply]
Hi Fabian, Thank you for reminding me of the link to the paper! Sorry that I didn't find the paper previously.
